apps/client/src/app/pages/resources/resources-page.html

```diff
@@ -1,258 +1,30 @@
-          <div class="media-body">
-            <h3 class="h5 mt-0">FIRE</h3>
-            <div class="mb-1">
-              FIRE is a movement that promotes saving and investing to achieve
-              financial independence and early retirement.
-            </div>
-            <div>
-              <a href="../en/blog/2023/07/exploring-the-path-to-fire">FIRE →</a>
-            </div>
-          </div>
-        </div>
-        <div class="mb-4 media">
-          <div class="media-body">
-            <h3 class="h5 mt-0">Inflation</h3>
-            <div class="mb-1">
-              Inflation is an increase of the general price level for goods and
-              services in an economy over a period of time.
-            </div>
-            <div>
-              <a
-                href="https://www.investopedia.com/terms/i/inflation.asp"
-                target="_blank"
-                >Inflation →</a
-              >
-            </div>
-          </div>
-        </div>
-        @if (hasPermissionForSubscription) {
-          <div class="mb-4 media">
-            <div class="media-body">
-              <h3 class="h5 mt-0">Personal Finance Tools</h3>
-              <div class="mb-1">
-                Personal finance tools are software applications that help
-                individuals manage their money, track expenses, set budgets,
-                monitor investments, and make informed financial decisions.
-              </div>
-              <div>
-                <a [routerLink]="routerLinkResourcesPersonalFinanceTools"
-                  >Personal Finance Tools →</a
-                >
-              </div>
-            </div>
-          </div>
-        }
-        <div class="mb-4 media">
-          <div class="media-body">
-            <h3 class="h5 mt-0">Stagflation</h3>
-            <div class="mb-1">
-              Stagflation describes a situation in which there is a stagnant
-              economy with high unemployment and high inflation.
-            </div>
-            <div>
-              <a
-                href="https://www.investopedia.com/terms/s/stagflation.asp"
-                target="_blank"
-                >Stagflation →</a
-              >
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<mat-tab-nav-panel #tabPanel class="flex-grow-1 overflow-auto">
+  <router-outlet></router-outlet>
+</mat-tab-nav-panel>
+
+<nav
+  mat-align-tabs="center"
+  mat-tab-nav-bar
+  [disablePagination]="true"
+  [tabPanel]="tabPanel"
+>
+  @for (tab of tabs; track tab) {
+    @if (tab.showCondition !== false) {
+      <a
+        #rla="routerLinkActive"
+        class="no-min-width px-3"
+        mat-tab-link
+        routerLinkActive
+        [active]="rla.isActive"
+        [routerLink]="tab.path"
+        [routerLinkActiveOptions]="{ exact: true }"
+      >
+        <ion-icon
+          [name]="tab.iconName"
+          [size]="deviceType === 'mobile' ? 'large' : 'small'"
```

Comment:
I think you need to define `deviceType`

(see `this.deviceType = this.deviceService.getDeviceInfo().deviceType;` in other page components)

Reply:
Hey sorry about that missed that line. I have added it in.